lib/trino-orc/src/main/java/io/trino/orc/metadata/statistics/BloomFilter.java

```diff
@@ -414,19 +413,19 @@ public static long hash64(Slice data)
             long k = 0;
             switch (data.length() - current) {
                 case 7:
-                    k ^= ((long) UnsafeSlice.getByteUnchecked(data, current + 6) & 0xff) << 48;
+                    k ^= ((long) data.getByteUnchecked(current + 6) & 0xff) << 48;
```

Comment:
`Slice.getByte()` and `Slice.getByteUnchecked()` should be equivalent now, no?

Reply:
`getByte` performs a bounds check on the Slice's view of the underlying byte[].